content/news/registration-dapc-24.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Registration DAPC 2024"
+date: 2024-06-12T09:00:00+02:00
+tags: ["DAPC 2024"]
+draft: false
+---
+Also in 2024, the CHipCie will organise the Delft Algorithm Programming Contest (DAPC). This event is free for all TU Delft
+students. The best teams will qualify to represent the Delft University of Technology on the [Benelux Algorithm Programming Contest](https://2023.bapc.eu/).
```

Comment:
The 2024 website is not ready yet, but we can just link to the "latest" version (which will be updated automatically once Groningen requests the DNS update):
```suggestion
students. The best teams will qualify to represent the Delft University of Technology on the [Benelux Algorithm Programming Contest](https://bapc.eu/).
```